scripts/get-version

```diff
@@ -1,76 +1,80 @@
-#!/bin/sh
+#!/bin/bash
+# shellcheck disable=SC2164
 
-# $1 - os name {linux|darwin|win}
-# $2 - [optional] raw version string "vX.Y-patchN-sha1". as from `git describe'
-#      (see below)
-
-set -eu
-#set -x
+# consistently name all builds, regardless of OS or desktop/mobile
+#
+# we do need to be able to create three digit (M.m.p) and four digit (M.m.p.c) version strings
+# default is VERSION_EXTENSION version - an argument of '4' or '3' gets you a digits only version string
+#
+# we hardcode a base version - this will rarely change
```

Comment:
I think this is just what we need for CICD builds - it makes sure that the version number is defined when the automatic build starts, and doesn't change after this.
One minor improvement could be to have the base version stored in a checked in file of its own - this way it will be trivial to get a version history by looking at the `git log` of this file.

Reply:
I'm not sure I understand that comment. Can you elaborate?

Reply:
I was referring to hardcoding the base version, which I think is the right thing to do for CICD releases.
But as a small improvement we could store the version base in a file like `subsurface-base-version` and then read it in the script as `$(cat subsurface-base-version)` - I can pull request this if you want.

Reply:
Given that it's stored in one place in one file... isn't that the better way to do it? That's the one file that you mess with when you want to make changes with the version...
I mean I'm fine either way - I'm just not sure I see the benefit.